example/android/build.gradle

```
@@ -3,18 +3,18 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
```

I don't believe it's valid to have bulidToolsVersion at 28 while any other version is higher, and we've got targetSdkVersion/compileSdkVersion set to 30 here. buildToolsVersion should also go to 30 I think

Separately there are [reports in other modules I maintain](https://github.com/react-native-device-info/react-native-device-info/pull/1238#issuecomment-930683301) that the default ndkVersion is [no longer working in certain cases](https://github.com/actions/virtual-environments/issues/4144), and it can fail in other ways, so it's better to pin it - I've added a pin here to current template version https://github.com/facebook/react-native/blob/51bf55794899284e1c465d346a3f6ebd8a485da2/template/android/build.gradle#L9

```suggestion
        buildToolsVersion = "30.0.2"
        ndkVersion = "21.4.7075529"
```